CoreFarms.cs

```diff
@@ -296,6 +296,10 @@ public void IcestormArena(int level = 100, bool rankUpClass = false)
         Bot.Options.AggroMonsters = OptionRestore;
         Core.SavedState(false);
         Core.ToggleAggro(false);
+        if (!rankUpClass)
```

Comment:
line 236 ?

Reply:
The script doesn’t disable the class boost by the end of class rank up, it will stay active until you ran out of boost. I think it’s a good idea to deactivate it?

Reply:
Wait I think I did it wrong